lib/manageiq_foreman/lib/highline_helper.rb

```diff
@@ -0,0 +1,53 @@
+require 'highline/import'
+require 'highline/system_extensions'
+include HighLine::SystemExtensions
+
+# stolen from appliance_console/prompts.rb
+module HighlineHelper
```

Comment:
What's this for?  I don't see it pulled in anywhere, and I'm not sure how it's used.